db/schema.rb

```diff
@@ -39,12 +39,9 @@
     t.string "address"
     t.float "lat"
     t.float "long"
-    t.string "contact_name_1"
-    t.string "contact_number_1"
-    t.string "email_1"
-    t.string "contact_name_2"
-    t.string "contact_number_2"
-    t.string "email_2"
```

Comment:
We probably shouldn't remove the two sets of contact info, as the garden model is set up for two sets. You probably just accidentally committed an old version of this file.

Reply:
Yeah, it was weird. When I rebased my local chase branch onto my master, it modified the schema to be the version in my chase branch instead of the more recent version in master. Kohei and I seemed to have fixed that